src/solverpip.ts

```diff
@@ -219,6 +232,30 @@ export async function solvePip(
   return pipSolvedPackages;
 }
 
+async function getPipPackageName(
+  installedPackage: string,
+  logger?: ILogger
+): Promise<string> {
+  let result = '';
+  try {
+    const url =
+      'https://raw.githubusercontent.com/prefix-dev/parselmouth/main/files/compressed_mapping.json';
```

Comment:
We can't fetch this JSON file like this, this will fail with CORS issues.

We will need to either store this JSON file somewhere where CORS is not an issue, or hardcode the JSON content in the mambajs code.

Could we solve this issue in a separate PR to make this PR more focused?

Reply:
Actually I'm wrong, that server does implement the proper headers so we don't have CORS issues.

It only has a limit on the usage, based of the IP of the request. 

We can go ahead with this for now I guess!